Scrolling for getting an item and adding multiple remove item attempts

estes-sj opened the pull request:
Closes #46 and potentially closes #50.

**Summary**
- Addressed TODO in `_get_alexa_list_item_element` to fully scroll through list.
  - In large lists (15+), removing items at the end of the list would not be found because of similar scrolling issues. This would cause the HA component to crash and require reload.
- Updated `remove_alexa_list_item()` to attempt multiple times when removing an item
  - In large lists (15+). removing an item located towards the end of the list would sometimes not be found, even when scrolling. On HA, the crossed off item would just be added back to the list. There may be a better way with selenium to handle this but I didn't notice it taking more than 2 attempts to remove an item at the end of a 20 item list.

madmachinations commented:
Thanks for this, saved me a job and it's greatly appreciated haha. Assuming that one comment is about something redundant and inconsequential then happy to merge. I'll wait for your reply @estes-sj

estes-sj commented:
> Assuming that one comment is about something redundant and inconsequential

Not sure what you mean. Are you referring to the comment on line 261 in `remove_alexa_list_item()`?

saberstop commented:
Thanks for this work! Appreciate the time and effort.

saberstop commented:
@madmachinations would you be able to clarify your question? Don't wanna bug you, but it would be nice to get this merged

madmachinations commented:
@estes-sj please look nearer the top on this pull request, it says I have marked pending changes. Can you see that?

I'm not talking about comments _in_ the code, I'm talking about comments I have written on this pull request.

Because I'm not sure about a couple of lines of code you have written, which appear to do nothing 😅

I'm asking if those seemingly redundant lines are simply leftovers from your development process which can be removed, or if that array is supposed to be used somewhere and is just not hooked up to anything by accident.

estes-sj commented:
@madmachinations Not sure why but when I went to the commits and files changed tab, I didn't see any comments left on it for review. Was "Finish Review" clicked afterwards? (Not doubting that I missed something on my end.)

Regardless, I removed the unneeded array and updated the commit.

madmachinations commented:
Thanks @estes-sj that's awesome, very much appreciated! I will merge it 🥳

madmachinations commented:
New build is deployed and available
